src/core_modules/capture-core/components/DataEntry/actions/dataEntryLoad.utils.js

```diff
@@ -88,7 +88,7 @@ export function getDataEntryNotes(
     const notes = clientValuesForDataEntry.notes || [];
     return notes.map((note, index) => ({
         ...note,
-        storedAt: convertListValue(note.storedAt, dataElementTypes.DATETIME),
+        storedAt: convertServerToClient(note.storedAt, dataElementTypes.DATETIME),
```

Comment:
For DATETIME the serverToClient converter doesn't work as expected currently. But, trying to debug the code, it seems to me that we should just remove this conversion anyway. Let's fix the other comment first and then we can verify.

Reply:
What do you mean by not working as expected ?